cpp/log/cert_submission_handler.cc

```diff
@@ -112,15 +134,15 @@ CertSubmissionHandler::ProcessPreCertSubmission(PreCertChain* chain,
   string der_cert;
   // Nothing should fail anymore as we have validated the chain.
   if (chain->LeafCert()->DerEncoding(&der_cert) != Cert::TRUE)
-    return INTERNAL_ERROR;
+    return Status(util::error::INTERNAL, "could not DER-encode the chain");
```

Comment:
I wouldn't really bother, since `DerEncoding` is going to return a `util::Status` soon, so it'll just be a "classic" `Status status(thing()); if (!status.ok) return status;`.